lib/scala/pkg/src/main/java/org/enso/pkg/archive/TarArchiveBuildingFileVisitor.java

```diff
@@ -0,0 +1,97 @@
+package org.enso.pkg.archive;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.*;
+import java.nio.file.attribute.BasicFileAttributes;
+import org.apache.commons.compress.archivers.tar.TarArchiveEntry;
+import org.apache.commons.compress.archivers.tar.TarArchiveOutputStream;
+import org.apache.commons.compress.archivers.tar.TarConstants;
+import org.enso.pkg.Package$;
+
+/** A file visitor that builds a tar archive out of visited files. */
+public final class TarArchiveBuildingFileVisitor implements FileVisitor<Path> {
```

Comment:
I'd expect this to be an implementation detail class and I'd keep it _package private_.

Reply:
Made package private